documentation/modules/ROOT/pages/1setup.adoc

```diff
@@ -38,7 +38,7 @@ You will need in this tutorial:
 ====
 If you want to validate if everything is installed at once, just open a terminal and run:
 
-`curl -sL https://git.io/_has | HAS_ALLOW_UNSAFE=y bash -s`
+`curl -sL https://git.io/_has | HAS_ALLOW_UNSAFE=y bash -s minishift docker oc kubectl git mvn curl stern siege`
```

Comment:
if you are running from root directory this is not needed since it takes the `.hasrc` file. https://github.com/redhat-developer-demos/istio-tutorial/blob/master/.hasrc

Reply:
ah - thanks.  Good to know.  Much appreciated.

Reply:
If you can remove this change then I'll merge the PR :)